services/blockchain-connector/methods/pos.js

```diff
@@ -20,6 +20,7 @@ const {
 	getPosLockedRewards,
 	getStaker,
 	getPosPendingUnlocks,
+	getClaimableRewards,
```

Comment:
Can we please rename `getClaimableRewards` to `getPosClaimableRewards` across the entire implementation to make it evident that this information comes from the `PoS` module and not the `reward/dynamicReward`, especially for someone new looking into the codebase?